config/locales/es.yml

```
@@ -63,77 +63,77 @@ es:
         password: Contraseña
     errors:
       messages:
-        unpwn:
-        blocked:
+        unpwn: ha aparecido con anterioridad en una filtración de datos y no se debería usar
+        blocked: "El dominio '%{domain}' ha sido bloqueado por spam. Por favor utiliza un email personal válido."
       models:
         ownership:
           attributes:
             user_id:
-              already_confirmed:
-              already_invited:
+              already_confirmed: ya es propietario de esta gema
+              already_invited: ya ha sido invitado a esta gema
         version:
           attributes:
             gem_full_name:
-              taken:
+              taken: "%{value} ya existe"
             full_name:
-              taken:
+              taken: "%{value} ya existe"
     models:
-      user:
+      user: Usuario
   activemodel:
     errors:
       models:
         oidc/api_key_permissions:
           attributes:
             valid_for:
-              inclusion:
+              inclusion: "%{value} segundos debe estar entre 5 minutos (300 segundos) y 1 día (86.400 segundos)"
   api_keys:
     create:
-      success:
-      invalid_gem:
+      success: "Nueva clave de API creada"
+      invalid_gem: La gema seleccionada no se puede incluir en esta clave
     destroy:
-      success:
+      success: "Clave de API eliminada con éxito: %{name}"
     index:
-      api_keys:
-      name:
-      scopes:
-      gem:
-      age:
-      last_access:
-      action:
-      delete:
-      confirm:
-      confirm_all:
-      new_key:
-      index_rubygems:
-      push_rubygem:
-      yank_rubygem:
-      add_owner:
-      remove_owner:
-      access_webhooks:
-      show_dashboard:
-      reset:
-      save_key:
-      mfa:
+      api_keys: Claves de API
+      name: Nombre
+      scopes: Alcances
+      gem: Gema
+      age: Edad
+      last_access: Último accesso
+      action: Acción
+      delete: Borrar
+      confirm: Se va a invalidar la clave de API. ¿Quieres confirmar?
```

```suggestion
      confirm: Se va a invalidar la clave de API. ¿Estás seguro?
```
In the english version it asks are you sure? Not if you want to confirm
`The API key will become invalid. Are you sure?`